image/embed_charts_test.go

```diff
@@ -99,24 +98,14 @@ func (s *embedTestSuite) TestSecuredClusterChartShouldIgnoreFeatureFlagValuesOnR
 // secured cluster.
 func (s *embedTestSuite) TestLoadSecuredClusterScanner() {
 	testCases := map[string]struct {
-		kubectlOutput                 bool
-		enableLocalScannerFeatureFlag bool
-		expectScannerFilesExist       bool
+		kubectlOutput           bool
+		expectScannerFilesExist bool
 	}{
-		"with feature flag is disabled should not contain scanner manifests": {
-			enableLocalScannerFeatureFlag: false,
-			expectScannerFilesExist:       false,
-		},
-		"with feature flag enabled contains scanner manifests ": {
-			enableLocalScannerFeatureFlag: true,
-			expectScannerFilesExist:       true,
-		},
-		"in kubectl output and feature flag enabled does not contain scanner manifests": {
-			kubectlOutput:                 true,
-			enableLocalScannerFeatureFlag: true,
-			expectScannerFilesExist:       false,
+
+		"contains scanner manifests ": {
```

Comment:
```suggestion
		"contains scanner manifests": {
```
Might also make sense to explicitly set `kubectlOutput` to `false` for readability.